rust/android/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "rust"
-version = "1.0.0"
-authors = ["Aditya Kulkarni <aditya@zecwallet.co>"]
+version = "1.0.1"
+authors = ["Zingolabs <zingo@zingolabs.com>"]
```

Comment:
Let's append, rather than replace.

May as well try edition 2021 while we're at it!